packages/web/src/components/common/select-token/SelectToken.tsx

```diff
@@ -35,6 +40,7 @@ const SelectToken: React.FC<SelectTokenProps> = ({
 
   const onClickClose = useCallback(() => {
     close();
+    callback?.(true);
```

Comment:
The callback function has already been executed in the parent container. 
Does this need to be called here as well?

Reply:
Updated